frontend/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -235,14 +235,25 @@ class Multiselect extends React.PureComponent<Props, State> {
                   paddingRight: ".5rem",
                 }),
               },
-
               ControlContainer: {
-                style: () => ({
+                style: ({ $isFocused }) => ({
                   // Baseweb requires long-hand props, short-hand leads to weird bugs & warnings.
                   borderLeftWidth: "1px",
                   borderRightWidth: "1px",
                   borderTopWidth: "1px",
                   borderBottomWidth: "1px",
+                  borderRightColor: $isFocused
```

Comment:
I think the focus color needs to be fixed by using the new theme property instead of applying it directly here (see my comment in the PR).